### PR TITLE
Handle race condition CYP-978

### PR DESCRIPTION
**Pull request by abhishek-lambda**

### CODE REVIEW

File: commands/generate_reports.js [hunk 1]
```diff
@@ -93,8 +97,10 @@ function download_artefact(
           reject("Could not download artefacts for test id " + test_id);
         }
       } else if (error.request) {
+        console.log("Got error: error in request",error.toJSON());
```
Comment (HRanjan-11) [inline commands/generate_reports.js:100]: remove this line already logged below.

File: commands/generate_reports.js [hunk 2]
```diff
@@ -217,6 +223,7 @@ function generate_report(args) {
         const downloadPromises = [];
 
         for (i = 0; i < build_info["data"].length; i++) {
+          console.log("Downloading artefacts for ", build_info["data"][i]["test_id"]);
```
Comment (HRanjan-11) [inline commands/generate_reports.js:226]: please remove this also

**Review by HRanjan-11 — APPROVED** [review]

File: commands/generate_reports.js (hunk 1 above)
Comment (abhishek-lambda) [reply commands/generate_reports.js:100]: done

File: commands/generate_reports.js (hunk 2 above)
Comment (abhishek-lambda) [reply commands/generate_reports.js:226]: done

**Review by HRanjan-11 — APPROVED** [review]

**Pull request merged**